packages/message-kit/src/lib/runner.ts

```
@@ -91,20 +95,25 @@ export default async function run(handler: Handler, config?: Config) {
+    const isAdmin =
       group &&
-      !group?.isAdmin(sender.inboxId) &&
-      !group?.isSuperAdmin(sender.inboxId)
-        ? false
-        : true;
+      (group?.admins.includes(sender.inboxId) ||
+        group?.superAdmins.includes(sender.inboxId))
+        ? true
+        : false;
 
+    const isAdminOrPass = isAdminCommand && isAdmin ? true : true;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.

(lint/complexity/noUselessTernary)